apps/minimals/qt/qtminimum.cpp

```diff
@@ -88,6 +92,38 @@ int main(int argc, char** argv) {
         },
         1000);
 
+    TCLAP::ValueArg<int> fullscreenArg(
```

Comment:
Should be a `TCLAP::SwitchArg`

See for example
https://github.com/inviwo/inviwo/blob/5e6f46832db04a4f160e464d7de29b5be8f9d70c/include/inviwo/qt/editor/inviwomainwindow.h#L251-L252

Reply:
OK, forgot this :). All fixed now.